addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -215,6 +215,7 @@ export class ComposerTextInput extends Component {
         }
         this._textareaLastInputValue = this._textareaRef.el.value;
         this._updateHeight();
+        this.composerView.removeSuggestions();
```

Comment:
It's no good having logic in component, but this one is too cryptic.
`onInput` happens in most user interactions on the composer text input. The other LOCs make sense (save in store, adjust height), but this "remove Suggestions" is off.

Why do you need it? What's the problem? If there's a problem, it's definitely solved in another way.